WpfAnimatedGif/ImageAnimationController.cs

```diff
@@ -15,14 +15,14 @@ public class ImageAnimationController : IDisposable
 
         static ImageAnimationController()
         {
-            _sourceDescriptor = DependencyPropertyDescriptor.FromProperty(Image.SourceProperty, typeof (Image));
+            _sourceDescriptor = DependencyPropertyDescriptor.FromProperty(Image.SourceProperty, typeof(Image));
         }
 
         private readonly Image _image;
         private readonly ObjectAnimationUsingKeyFrames _animation;
         private readonly AnimationClock _clock;
         private readonly ClockController _clockController;
-        
+
```

Comment:
When you submit a PR, try not to change the formatting. In this case it's ok (this whitespace had nothing to do here), but not everyone use the same formatting style. As a general rule, when you contribute to a project, you should stick to the style in use in this project, and absolutely avoid the [wall of pink](http://www.hanselman.com/blog/YoureJustAnotherCarriageReturnLineFeedInTheWall.aspx) :wink: